### DISCUSSION TITLE
relationship field on chrome linux does not allow typing

### DISCUSSION THREAD

**Discussion by geraldodev** [Budibase/budibase#7044] [General]
Hi,

I was trying budibase yesterday and I've noticed that the select widget of the relationship field does not allow typing .
It's important to type on these fields granted that the relationship table may be big and using the mouse is not productive on these scenarios.

Regards,

Geraldo Lopes de Souza

**Comment by melohagan** [comment]
Hi @geraldodev 

Have you checked the **Autocomplete** checkbox? 
<img width="259" alt="Screenshot 2022-08-04 at 17 03 59" src="https://user-images.githubusercontent.com/101575380/182894552-0bea4d93-0e55-432a-b831-09233534d419.png">
